TCMSApp/src/client/app/tests/tests-create.html

```diff
@@ -22,9 +22,7 @@ <h1 class="text-center">Create New Test Case</h1>
               <div class="form-group">
                 <label for="caseSuite">Suite:</label>
                 <select class="form-control" id="caseSuite" name="caseSuite">
-                  <option>Suite 1</option>
-                  <option>Suite 2</option>
-                  <option>Suite 3</option>
+                  <option>{{ ::vmCase.currentSuite.suiteName }}</option>
                   <option>None</option>
```

Comment:
if we want test case without suite we name separate suite as: "tests without suites".
Not kidding. it seems we agreed on that